apps/web/src/routes/_content/resources/podcasts/$podcastId.tsx

```diff
@@ -80,37 +94,92 @@ function Podcast() {
-                </div>
 
-                <div className="mt-2 text-blue-700">
-                  <span className="text-xs font-light italic">
-                    {t('book.topicsAddressed')}
-                  </span>
-                  {podcast?.tags.map((object, i) => (
-                    <span key={i}>
-                      {i > 0 && ', '}
-                      {object.toUpperCase()}
+                  <div className="flex items-center flex-wrap">
+                    <span className="text-white body-14px-medium md:label-medium-med-16px pr-1">
+                      {t('words.writtenByPodcasts')}
                     </span>
-                  ))}
+                    <span className="text-white body-14px md:body-16px">
+                      {podcast.host}
+                    </span>
+                  </div>
                 </div>
+
                 {isScreenMd && displayAbstract()}
               </div>
             </div>
+
             {!isScreenMd && displayAbstract()}
           </Card>
         </div>
       )}
+
+      <section className="mt-8 lg:mt-[100px]">
+        <h3 className="label-medium-med-16px md:title-large-24px font-medium leading-none md:leading-[116%] text-white mb-5 md:mb-10">
+          {t('resources.pageSubtitlePodcast')}
+        </h3>
+        <Carousel>
+          <CarouselContent>
+            {isFetchedSuggested ? (
+              suggestedPodcasts?.map((suggestedPodcast) => (
+                <CarouselItem
+                  key={suggestedPodcast.id}
+                  className="text-white basis-1/2 md:basis-1/2 lg:basis-1/4 relative bg-gradient-to-r max-w-[280px] max-h-[350px] rounded-[22px]"
+                >
+                  <Link
+                    to={'/resources/podcasts/$podcastId'}
+                    params={{
+                      podcastId: suggestedPodcast.id.toString(),
+                    }}
+                  >
+                    <div className="relative h-full">
+                      <img
+                        className="max-h-72 sm:max-h-96 size-full object-cover rounded-[10px]"
+                        alt={suggestedPodcast.name}
+                        src={suggestedPodcast.logo}
+                      />
+                      <div
+                        className="absolute inset-0 -bottom-px rounded-[10px]"
+                        style={{
+                          background: `linear-gradient(360deg, rgba(40, 33, 33, 0.90) 10%, rgba(0, 0, 0, 0.00) 60%),
+                          linear-gradient(0deg, rgba(57, 53, 49, 0.20) 0%, rgba(57, 53, 49, 0.20) 100%)`,
+                          backgroundSize: '153.647% 100%',
+                          backgroundPosition: '-5.216px 0px',
+                          backgroundRepeat: 'no-repeat',
+                        }}
+                      />
+                    </div>
+
+                    <h3 className="absolute px-2 lg:px-4 body-14px lg:title-large-24px pb-1 lg:pb-5 bottom-px w-[120px] lg:w-[200px] truncate">
```

Comment:
Shouldn't be truncated on first line (keep 2-3 lines instead), you can use line clamp